crates/egui_kittest/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "egui_kittest"
+edition.workspace = true
+license.workspace = true
+rust-version.workspace = true
+version.workspace = true
```

Comment:
This lacks a lot of stuff, like:

* authors
* description
* readme
* categories
* keywords
* include

See any other `Cargo.toml` for more